swift/apple/FirezoneNetworkExtension/ConfigurationManager.swift

```diff
@@ -80,7 +80,17 @@ class ConfigurationManager {
   }
 
   @objc private func handleUserDefaultsChanged(_ notification: Notification) {
-    self.managedDict = userDefaults.dictionary(forKey: managedDictKey) ?? [:]
+    let newManagedDict = (userDefaults.dictionary(forKey: managedDictKey) ?? [:]) as [String: Any?]
+
+    // NSDictionary conforms to Equatable
+    if (managedDict as NSDictionary) != (newManagedDict as NSDictionary) {
```

Comment:
```suggestion
    if (self.managedDict as NSDictionary) != (newManagedDict as NSDictionary) {
```

Perhaps a tad easier to understand?

Reply:
Yeah we need consistent styling. I've been using the convention of only adding `self` when writing to the variable.